finetune/encoding/input_encoder.py

```diff
@@ -246,7 +246,7 @@ def tokenize_context(context, encoded_output, config):
                 current_char_loc += 1
                 if current_char_loc >= len(context_by_char_loc):
                     raise ValueError("Context cannot be fully matched as it appears to not cover the end of the sequence for token {}".format(token))
-            if token.strip() not in context_by_char_loc[current_char_loc][2]:
+            if context_by_char_loc[current_char_loc][2] and token.strip() not in context_by_char_loc[current_char_loc][2]:
```

Comment:
Wouldn't this still raise an error if we did an `in` check on c.get('text') and c.get('text') returned None?

Reply:
It doesn't get evaluated if `context_by_char_loc[current_char_loc][2]` is None because of the lazy and